### ISSUE TITLE
Perf try jobs with V8 patches don't have results (and hidden errors)

### ISSUE THREAD

**Issue by natorion** [catapult-project/catapult#3969]
We wanted to use the feature to [run Chromium perf tryjobs with V8 patches](https://docs.google.com/document/d/1ThZ9-Ik-Et4PC6LfnB-P4Jz43n0cSZhfuEsGCNxEluQ/edit). Here is the associated CL: https://chromium-review.googlesource.com/c/v8/v8/+/707253.

It seems that everything for the relevant platforms (e.g. Linux) worked fine like in this result: https://build.chromium.org/p/tryserver.chromium.perf/builders/linux_perf_bisect/builds/7686. According to [the perf try job documentation](https://chromium.googlesource.com/chromium/src/+/master/docs/speed/perf_trybots.md) there should be a results html. There is no mention about that anywhere on the bot result though. There is also no other obvious step that could link to the html file. Where is it located?

Another interesting thing: Some errors are hidden, see the attached screenshot.
![image](https://user-images.githubusercontent.com/7190422/31483614-65e545b8-af2d-11e7-8b45-324a4fef0412.png)

**Comment by natorion** [comment]
@nedn

**Comment by anniesullie** [comment]
@simonhatch 

Also @dave-2 will this use case work in pinpoint?

**Comment by simonhatch** [comment]
/usr/bin/python /b/c/b/linux_perf_bisect/src/tools/perf/run_benchmark --browser=release speedometer2 octane jetstream v8.browsing_desktop v8.browsing_mobile --verbose --output-format=html --upload-bucket=public --upload-results --results-label=TOT
Usage: run_benchmark run benchmark_name [page_set] [<options>]
run_benchmark: error: Too many arguments.


Looks like you tried to specify multiple benchmarks at once? This should work fine if you redo but only with 1 benchmark per perf try job.

**Comment by dave-2** [comment]
@anniesullie Yes. The patch will need to be in Gerrit. I haven't tested it though, so I'm not sure bot_update/gclient supports it, but I assume it would.
